File: 105.c

/**
 * @description: 记负均正Ⅱ
 * @param {type} 
 * @return: 
 */
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>

int count;
int sum;
int testnum;
int temp;
int zhengshucount;

int main(void)
{
    sum = 0;
    count = 0;
    zhengshucount = 0;

    while(scanf("%d", &temp) != EOF)
    {
        if(temp < 0)
        {
            count++;
        }
        else
        {
            zhengshucount++;
            sum += temp;
        }
    }
    printf("%d\n%.1f\n", count, (float)((float)sum/(zhengshucount)));

    return 0;
}

File: 105.py
'''
@description: 记负均正Ⅱ
@param {type} 
@return: 
'''
while True:
    try:
        inlist=list(map(int,input().split()))
        fushu=[]
        zhengshu=[]
        for i in inlist:
            if i<0:
                fushu.append(i)
            else:
                zhengshu.append(i)
        print(len(fushu))
        zhengshu_len=len(zhengshu)
        sum=0
        if zhengshu_len==0:
            print('0.0')
        else:
            for j in zhengshu:
                sum+=j
            print(round((sum/zhengshu_len),1 ))
    except:
        break